logging/user-values-opensearch.yaml

```diff
@@ -1,8 +1,8 @@
-# Chart: https://github.com/opensearch-project/helm-charts/tree/main/charts/opensearch
```

Comment:
Evidently, this file was not updated when we did our last few updates; that's why the diff shows so many changes.